concepticondata/conceptlists/Mountain-1978-267.tsv

```diff
@@ -0,0 +1,268 @@
+Mountain-1978-267-118	118	perico, pihuicho	parakeet	132	886	PARAKEET
+Mountain-1978-267-119	119	gallinazo	buzzard	133	1865	BUZZARD
+Mountain-1978-267-120	120	pavo	curassow	134	315	CURASSOW
+Mountain-1978-267-121	121	lechuza	owl	135	735	OWL
+Mountain-1978-267-122	122	pava de monte	guan	136	437	GUAN
+Mountain-1978-267-123	123	gallina	hen	137	1514	HEN
+Mountain-1978-267-124	124	pescado	fish	138	227	FISH
+Mountain-1978-267-125	125	caribe	piranha	139	317	PIRANHA
+Mountain-1978-267-126	126	abeja	bee	140	665	BEE
+Mountain-1978-267-127	127	mosca	fly	141	1504	FLY (INSECT)
+Mountain-1978-267-128	128	pulga	flea	142	232	FLEA
+Mountain-1978-267-129	129	piojo	louse	143	1392	LOUSE
+Mountain-1978-267-130	130	zancudo	mosquito	144	1509	MOSQUITO
+Mountain-1978-267-131	131	comején, hormiga blanca	termite	145	883	TERMITE
+Mountain-1978-267-132	132	hormiga	ant	146	587	ANT
+Mountain-1978-267-133	133	araña	spider	147	843	SPIDER
+Mountain-1978-267-134	134	nigua pique	chigger	148	438	CHIGGOE (JIGGER FLEA)
+Mountain-1978-267-135	135	monte	bush	149	1683	BUSH
+Mountain-1978-267-136	136	pajonal, sabana	open grassland	150	260	SAVANNA
+Mountain-1978-267-137	137	cerro, loma	hill	151	614	HILL
+Mountain-1978-267-138	138	árbol	tree	152	906	TREE
+Mountain-1978-267-139	139	hoja	leaf	153	628	LEAF
+Mountain-1978-267-140	140	flor	flower	155	239	FLOWER
+Mountain-1978-267-141	141	fruta	fruit	156	1507	FRUIT
+Mountain-1978-267-142	142	savia	sap	157	345	SAP
+Mountain-1978-267-143	143	raíz	root	158	670	ROOT
+Mountain-1978-267-144	144	hierba	grass	161	606	GRASS
+Mountain-1978-267-145	145	maíz	corn, maize	162	506	MAIZE
+Mountain-1978-267-146	146	yuca	manioc	163	927	MANIOC
+Mountain-1978-267-147	147	harina de yuca	manioc flour	164	324	MANIOC FLOUR
+Mountain-1978-267-148	148	tabaco	tobacco	165	974	TOBACCO
+Mountain-1978-267-149	149	algodón	cotton	166	1850	COTTON
+Mountain-1978-267-150	150	calabazo poro	gourd	167	411	GOURD
+Mountain-1978-267-151	151	ñame	yam	168	410	YAM
+Mountain-1978-267-152	152	camote	sweet potato	169	159	SWEET POTATO
+Mountain-1978-267-153	153	ají	chili pepper	171	919	CHILI PEPPER
+Mountain-1978-267-154	154	coca	coca	172	137	COCA
+Mountain-1978-267-155	155	ayahuasca, yajé	Banisterium	173	276	BANISTERIUM
+Mountain-1978-267-156	156	plátano	plantain	174	399	PLANTAIN
+Mountain-1978-267-157	157	chonta	chonta palm	175	440	CHONTA PALM
+Mountain-1978-267-158	158	palma mirití	miriti palm	177		
+Mountain-1978-267-159	159	caña brava	cane	178	659	REED
+Mountain-1978-267-160	160	sal	salt	179	1274	SALT
+Mountain-1978-267-161	161	chicha, masato	chicha	180	441	CHICHA
+Mountain-1978-267-162	162	uno	one	181	1493	ONE
+Mountain-1978-267-163	163	dos	two	182	1498	TWO
+Mountain-1978-267-164	164	sonajero, maraca	rattle	193	300	RATTLE
+Mountain-1978-267-165	165	tambor	drum	194	908	DRUM
+Mountain-1978-267-166	166	orejeras, aretes	ear ornament	196		
+Mountain-1978-267-167	167	yo	I	200	1209	I
+Mountain-1978-267-168	168	tú	thou	201	1215	THOU
+Mountain-1978-267-169	169	nosotros	we	205	1212	WE
```

Comment:
I think there is a mishap in Huber-1992-375-199, namely the English translation and the mapping.

Reply:
True. But Huber is in Lexibank, so I'm not sure whether this should be updated here and now. @chrzyki what do you think?

Reply:
Yes, looks like a problem in the Huber list. @CarolinHu - we'd update the Huber list here in Concepticon and then re-run the lexibank pipeline for the dataset itself.

Reply:
Since this was only one line, I added the change to Huber with the latest commit. Should I open an issue in Lexibank as well or is it enough if I notify you here @chrzyki?

Reply:
All good, thanks. I'm taking note. :)